### ISSUE TITLE
check-line-alignment with "always" option breaks if there is no @param or @return type

### ISSUE THREAD

**Issue by TGCrystal** [gajus/eslint-plugin-jsdoc#891]
The `check-line-alignment` rule, with the `always` option, gives an error when there is only one space between the `@param` tag and the parameter name if there is no type definition in between. The same issue occurs with the `@return` tag in this context. The error goes away if there are instead 2 spaces between the tag and the next word.
I noticed issue #714 is similar, but that applies to the `never` option and only the `@param` tag.

## Expected behavior
No errors are given.
The two spaces seems to be there for the type definition to go between, but in a Typescript project the jsdoc type definitions are redundant and unnecessary.

## Actual behavior
The following error is given for either test case below:
```
Expected JSDoc block lines to be aligned  jsdoc/check-line-alignment
```

## ESLint Config

```js
{
    "plugins": [
        "jsdoc"
    ],
    "rules": {
        "jsdoc/check-line-alignment": [
            2,
            "always"
        ]
    }
}
```

## ESLint sample

```js
/**
 * Function description.
 *
 * @param lorem Description.
 * @param sit   Description multi words.
 */
const fn = ( lorem, sit ) => {}
```

```js
/**
 * Function description.
 *
 * @return Return description.
 */
const fn = () => {}
```

## Environment

- Node version: 16.13.2
- ESLint version 8.17.0
- `eslint-plugin-jsdoc` version: 39.3.2
